src/main/java/com/rocketden/main/dto/room/JoinRoomMapper.java

```diff
@@ -0,0 +1,13 @@
+package com.rocketden.main.dto.room;
```

Comment:
Note on the whole file: I don't think we need a separate `Mapper` class for each class (`JoinRoomResponse` and `CreateRoomResponse`). I think these could be methods in a `Utility.java` file within a `util` folder, or a separate `Mapper` class that has all of these separate methods.

Reply:
Merged the two files into a single `RoomMapper` class. As we add more mappings I think it might make sense to move it all into a single `utils` folder as you suggested.